packages/suite/src/views/wallet/coinmarket/common/CoinmarketOffers/CoinmarketOffersItem.tsx

```diff
@@ -145,7 +149,7 @@ const CoinmarketOffersItem = ({ quote }: CoinmarketOffersItemProps) => {
                         <StyledButton
                             isLoading={callInProgress}
                             isDisabled={!!quote.error}
-                            onClick={() => selectQuote(quote)}
+                            onClick={() => selectQuote(quote as any)}
```

Comment:
why any? should be fine without it

Reply:
Should be deprecated... https://github.com/trezor/trezor-suite/pull/12586/commits/4cd6f3b4120f82dd10dc267abe24f6206182c758